src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -675,8 +675,7 @@ void func_8088DD34(EnElf* this, PlayState* play) {
         !func_8088C804(&this->actor.world.pos, &refActor->actor.world.pos, 10.0f)) {
         Health_ChangeBy(play, 0x80);
         if (this->fairyFlags & 0x200) {
-            Magic_Add(play, ((void)0, gSaveContext.magicFillTarget) +
-                                (gSaveContext.save.playerData.isDoubleMagicAcquired + 1) * MAGIC_NORMAL_METER);
+            Magic_Add(play, MAGIC_ADD_TO_CAPACITY);
```

Comment:
I don't like the wording here, it could mean "FILL_TO_CAPACITY" or "ADD_ON_TOP_OF_CAPACITY", and the former is what it should be, right?

Reply:
It should be `FILL_TO_CAPACITY` but the reason I avoided that was to indicate that this should be for the `Magic_Add` function, maybe `MAGIC_ADD_TO_FILL_CAPACITY`? Although that sounds weird. Maybe `MAGIC_FILL_TO_CAPACITY` is fine?

Reply:
I think just commenting above the macro will have to do, it's very difficult to explain that it's filling the bar to capacity rather than adding capacity in some way otherwise.